plots/ab-screenshot/analyze.js

```diff
@@ -0,0 +1,211 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const args = require('yargs').argv;
+
+const Metrics = require('../../lighthouse-core/lib/traces/pwmetrics-events');
+
+const constants = require('../constants');
+const utils = require('../utils');
+
+/**
+ * Do a comparison analysis of two batches of runs. This helps you compare how a change in
+ * lighthouse can affect perf results in real-world sites.
+ *
+ * Example usage:
+ * node analyze.js -a ./out-first -b ./out-second
+ */
+function main() {
+  const outPathA = args.a;
+  const outPathB = args.b;
+
+  if (!utils.isDir(outPathA) || !utils.isDir(outPathB)) {
+    console.log('ERROR: Make sure both -a and -b point to valid paths'); // eslint-disable-line no-console
+    console.log('a: ', outPathA); // eslint-disable-line no-console
+    console.log('b: ', outPathB); // eslint-disable-line no-console
+    return;
+  }
+
+  const aggregatedScreenshots = {
+    data: aggregate(outPathA, outPathB),
+    a: outPathA,
+    b: outPathB
+  };
+
+  if (!utils.isDir(constants.OUT_PATH)) {
+    fs.mkdirSync(constants.OUT_PATH);
+  }
+  const outFilePath = path.resolve(constants.OUT_PATH, 'screenshotsComparison.js');
+  fs.writeFileSync(
+    outFilePath,
+    `var aggregatedScreenshots = ${JSON.stringify(aggregatedScreenshots, undefined, 2)}`
+  );
+  console.log('Wrote output to:', outFilePath); // eslint-disable-line no-console
+}
+
+main();
+
+/**
+ * Aggregates the results from two out paths.
+ * Note: only the first run for each site of each batch is used.
+ * @param {string} outPathA
+ * @param {string} outPathB
+ * @return {!AggregatedScreenshots}
+ */
+function aggregate(outPathA, outPathB) {
+  const results = [];
+
+  fs.readdirSync(outPathA).forEach(siteDir => {
+    const sitePathA = path.resolve(outPathA, siteDir);
+    const sitePathB = path.resolve(outPathB, siteDir);
+
+    // Skip a site if it's not in both batches.
+    if (!utils.isDir(sitePathB)) {
+      return;
+    }
+    const siteScreenshotsComparison = {
+      siteName: siteDir,
+      runA: analyzeSingleRunScreenshots(sitePathA),
+      runB: analyzeSingleRunScreenshots(sitePathB)
+    };
+    results.push(siteScreenshotsComparison);
+  });
+
+  return results;
+}
+
+/**
+ * Analyzes the screenshots for the first run of a particular site.
+ * @param {string} sitePath
+ * @return {!SingleRunScreenshots}
+ */
+function analyzeSingleRunScreenshots(sitePath) {
+  const runDir = sortRunFolders(fs.readdirSync(sitePath))[0];
+  const runPath = path.resolve(sitePath, runDir);
+  const lighthouseResultsPath = path.resolve(runPath, constants.LIGHTHOUSE_RESULTS_FILENAME);
+  const lighthouseResults = JSON.parse(fs.readFileSync(lighthouseResultsPath));
+
+  const fcpTiming = getTiming('ttfcp');
+  const fmpTiming = getTiming('ttfmp');
+  const vc100Timing = getTiming('vc100');
```

Comment:
can we get 85% too? not sure if it's in pwmetrics yet but we should since we'll be switching up TTI soon enough :)